P/PortMidi/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "PortMidi"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/PortMidi/portmidi.git", "928520f7b79f371854387cb480b4e3b5bf5dac95")
+]
```

Comment:
We usually prefer to build stable releases of software, and not random commits, and the `version` variable should match the upstream version number: https://docs.binarybuilder.org/stable/building/#Version-number

Reply:
Thanks!

I changed it accordingly, using `v2.0.4` which is the latest stable tag release. (The software is pretty old anyway, not many new releases.)